### PR TITLE
Add new page on popular configuration options

### PR DESCRIPTION
**Pull request by johanmalm**

### CODE REVIEW

File: src/configuration.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+<theme>
+  <dropShadows>yes</dropShadows>
+</theme>
+```
+
+### Enable automatic window placement policy
+
+```
+<placement>
+ <policy>automatic</policy>
```
Comment (Consolatis) [inline src/configuration.md:19]: The other's seem to use 2 spaces indentation.

File: src/configuration.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+### Unbind alt + arrow
+
+...because they interfere with brower forward/backward binds
+
+```
+<keybind key="A-Right"/>
+<keybind key="A-Left"/>
+<keybind key="A-Up"/>
+<keybind key="A-Down"/>
+```
```
Comment (Consolatis) [inline src/configuration.md:119]: Should we move this way up?

File: src/configuration.md (hunk 1 above)
Comment (johanmalm) [reply src/configuration.md]: Done.

File: src/configuration.md (hunk 2 above)
Comment (johanmalm) [reply src/configuration.md]: Have moved to top of keybind section.

**Pull request merged**